plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -1028,7 +1040,7 @@ const getCartAndCheckoutFrontendConfig = ( options = {} ) => {
 			rules: [
 				{
 					test: /\.(j|t)sx?$/,
-					exclude: /node_modules/,
+					exclude: [ /build/, /node_modules/ ],
```

Comment:
As we are exporting multiple configuration here, we can try https://webpack.js.org/configuration/configuration-types/#parallelism.